operator/cmd/mesh/operator-remove.go

```diff
@@ -79,15 +82,24 @@ func operatorRemove(cmd *cobra.Command, args *RootArgs, orArgs *operatorRemoveAr
 	if installed && err != nil {
 		l.LogAndFatal(err)
 	}
-	if !installed {
+	if !installed && !orArgs.purge {
 		l.LogAndPrintf("Operator controller is not installed in %s namespace (no Deployment detected).", orArgs.operatorNamespace)
 		if !orArgs.force {
 			l.LogAndFatal("Aborting, use --force to override.")
 		}
 	}
 
+	// If the user is performing purge but also specified a revision, we should warn
```

Comment:
I was wondering about this case, and very much prefer this to be an error instead of proceeding. I think if both present, then an error message should be displayed and nothing should be done, considering this is a highly destructive operation.

Reply:
Updated.

Reply:
@my-git9 Thanks for updating the PR.